server/main.lua

```diff
@@ -109,9 +109,23 @@ RegisterNetEvent('qbx_jewelery:server:endcabinet', function()
     sharedConfig.vitrines[closestVitrine].isBusy = false
     startedVitrine[source] = nil
 
+    local customDropItems = {}
     for _ = 1, math.random(config.reward.minAmount, config.reward.maxAmount) do
         local RandomItem = config.reward.items[math.random(1, #config.reward.items)]
-        player.Functions.AddItem(RandomItem.name, math.random(RandomItem.min, RandomItem.max))
+        local quantity = math.random(RandomItem.min, RandomItem.max)
+
+        if exports.ox_inventory:CanCarryItem(source, RandomItem.name, quantity) then
+            exports.ox_inventory:AddItem(source, RandomItem.name, quantity)
+        else
+            if config.CreateCustomDrop then
+                customDropItems[#customDropItems+1] = {RandomItem.name, quantity}
+            end
```

Comment:
Maybe we should just remove the config option altogether and make the behavior that the items the player is unable to carry drop on the ground. Otherwise they'd just disappear, which doesn't sound like behavior a server would desire. Wdyt?

Reply:
I added it optional because not everyone wants things to fall on the floor, perhaps they simply want it to get lost if they can't carry it and that's it. At least in my city I do it that way so that another person involved in the robbery can collect it. As you prefer friend.

Reply:
I think we should remove the config option and just have that be the default behavior.

Reply:
> I think we should remove the config option and just have that be the default behavior.

Perfect, ill commit changes now.

Reply:
Done, take a look and let me know.